entity-framework/core/what-is-new/ef-core-7.0/plan.md

```diff
@@ -0,0 +1,342 @@
+
+### Breaking changes
+
+EF7 will contain a small number of [breaking changes](xref:core/what-is-new/ef-core-7.0/breaking-changes) as we continue to evolve both EF Core and the .NET platform. Our goal is to minimize breaking changes as much as possible.
+
+## Themes
+
+The large investments in EF7 will fall mainly under the following themes:
+
+- Highly requested features
+- NET platforms and ecosystem
+- Clear path forward from EF6
+- Performance
+
+Each of these themes is described in detail below. The high-level status of each theme can be tracked in the [EF Core biweekly updates](https://github.com/dotnet/efcore/issues/23884). Please comment on [GitHub Issue #26994](https://github.com/dotnet/efcore/issues/26994) with any feedback or suggestions.
+
+## Theme: Highly requested features
+
+As always, a major input into the [planning process](xref:core/what-is-new/release-planning) comes from [votes (👍) for features on GitHub](https://github.com/dotnet/efcore/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc). Based on these votes in addition to other factors, we plan to work on the following highly requested features for EF7.
+
+### JSON columns
+
+Tracked by [Issue #4021: Map JSON values stored in database to EF properties](https://github.com/dotnet/efcore/issues/4021)
+
+Value proposition: Save and query into JSON-based documents stored in relational database columns.
+
+This feature will introduce a common mechanism and patterns for JSON support that can be implemented by any database provider. We will work with the community to align existing implementations for [Npgsql](https://github.com/npgsql/efcore.pg) and [Pomelo MySQL](https://github.com/PomeloFoundation/Pomelo.EntityFrameworkCore.MySql), and also add support for SQL Server and SQLite.
+
+### Bulk updates
+
+Tracked by [Issue #795: Bulk (i.e. set-based) CUD operations (without loading data into memory)](https://github.com/dotnet/efcore/issues/795)
+
+Value proposition: Efficient, predicate-based updates for many database rows without loading data into memory.
+
+[Change tracking](xref:core/change-tracking/index) followed by [SaveChanges](xref:core/saving/index) is the primary mechanism in EF Core for inserting, updating, and deleting entities. This mechanism ensures that database operations are ordered to satisfy constraints, and that tracked entities are kept in sync with the changes made to the database. However, it requires a database roundtrip to load entities into memory in order to create the appropriate database commands, followed by a database roundtrip to execute these commands.
+
+In contrast, bulk, or set-based, updates involve defining the changes that should be made to the database and then executing those changes without first loading entities into memory. This can be significantly faster than tracked updates, especially when the same modification must be applied to many different entities/rows.
```

Comment:
```suggestion
In contrast, bulk, or set-based, updates involve defining the changes that should be made to the database and then executing those changes without first loading entities into memory. In SQL, bulk updates are applied by sending statements such as `UPDATE [Blogs] SET [Rating] = [Rating] + 1`. This can be significantly faster than tracked updates, especially when the same modification must be applied to many different entities/rows.
```